code/game/objects/items/devices/ioncannon_remote.dm

```diff
@@ -10,12 +10,21 @@
 	var/cooldown = 0
 	mech_flags = MECH_SCAN_FAIL
 
+/obj/item/device/loic_remote/New()
+	processing_objects.Add(src)
+
 /obj/item/device/loic_remote/process()
 	update_icon()
 
 /obj/item/device/loic_remote/update_icon()
-	icon_state = "batterer[cooldown-world.time<0 ? "" : "burnt"]"
-
+	icon_state = "batterer[cooldown-world.time < 0 ? "" : "burnt"]"
+
+/obj/item/device/loic_remote/examine(mob/user)
+	..()
+	if(cooldown-world.time < 0)
+		to_chat(user, "<span class='notice'>It is ready to fire.</span>") 
+	else
+		to_chat(user, "<span class='notice'>The Low Orbit Ion Cannon can fire again in [cooldown-world.time].</span>") 
```

Comment:
Please convert to seconds! world time is in deciseconds.

For example, say this had 27 seconds remaining, it would say "...fire again in 270."

instead, use `[altFormatTimeDuration(cooldown-world.time)]`